.github/workflows/deploy-staging.yml

```diff
@@ -117,6 +118,7 @@ jobs:
     needs: files-changed
     with:
       deploy_env: "staging"
+      location_services_cdk_stack: ${{ vars.LOCATION_SERVICES_STACK_NAME_STAGING }}
```

Comment:
could make this optional when deploying infra for the widget, eh?

Reply:
💯 - was rushing and didn't realize this was deploying the widget 🤦